### PR TITLE
Update logging when podman or other preset in use

### PR DESCRIPTION
**Pull request by praveenkumar**
With this patch you shouldn't get following logging message in case of microshift bundle.
```
INFO Loading bundle: crc_microshift_hyperv_4.12.6_amd64...
INFO Creating CRC VM for Podman ...
```

**Fixes:** Issue #3557

### CODE REVIEW

**Review by anjannath — APPROVED** [review]

**Review by cfergeau — COMMENTED** [review]
Correct spellings are OKD, MicroShift, OpenShift and Podman. I don't think it's correct to special case Podman so that we use the correct capitalization, while for the others we are fine with printing the preset name as a string which will be lower case.
You could add a `startConfig.Preset.ForDisplay()` helper or such to solve this.

**Comment by praveenkumar** [comment]
>Correct spellings are OKD, MicroShift, OpenShift and Podman. I don't think it's correct to special case Podman so that we use the correct capitalization, while for the others we are fine with printing the preset name as a string which will be lower case.

@cfergeau I changed `Podman` string also to use preset's String method.

> You could add a startConfig.Preset.ForDisplay() helper or such to solve this.

Is there any reason why don't we change the const string value for preset to have correct spellings?
```
const (
	Podman     Preset = "Podman"
	OpenShift  Preset = "OpenShift"
	OKD        Preset = "OKD"
	Microshift Preset = "MicroShift"
)
```

**Comment by cfergeau** [comment]
> Is there any reason why don't we change the const string value for preset to have correct spellings?

At this point, it would break users' configurations. A config file with `preset: "podman"` would no longer be valid as the preset is `Podman`. Same for existing scripts, ...
More generally, I prefer to keep the enum internal representation separate from its string/user-visible representation. "Separate" as in, it should be opaque to code using it in our code base. If the code needs a string, then I prefer an explicit function call to get it as a string.

File: pkg/crc/machine/start.go [hunk 1]
```diff
@@ -242,10 +242,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 			return nil, errors.Wrap(err, "Failed to ask for pull secret")
 		}
 
-		if crcBundleMetadata.IsOpenShift() {
-			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetOpenshiftVersion())
+		if crcBundleMetadata.IsPodman() {
+			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetPodmanVersion())
 		} else {
-			logging.Infof("Creating CRC VM for Podman %s...", crcBundleMetadata.GetPodmanVersion())
+			logging.Infof("Creating CRC VM for %s %s...", startConfig.Preset, crcBundleMetadata.GetOpenshiftVersion())
```
Comment (cfergeau) [inline pkg/crc/machine/start.go]: Untested for now, but something like this should work too:
https://github.com/cfergeau/crc/commit/fcd0de25f6e8bea6414798e73f127badcee84cd5

Comment (praveenkumar) [reply pkg/crc/machine/start.go]: tested and that works, cherry-picked this commit

**Review by cfergeau — APPROVED** [review]

**Pull request merged**